sequence_check_digit/models/ir_sequence.py

```diff
@@ -37,17 +37,17 @@ def check_check_digit_formula(self):
         try:
             self.get_next_char(0)
         except Exception:
-            raise ValidationError(_("Format is not accepted"))
+            raise ValidationError(_("Format is not accepted")) from None
```

Comment:
Why did you add the from None?

Reply:
Due to the suggestion made by the pre-commit